dsgrid/config/date_time_dimension_config.py

```diff
@@ -0,0 +1,102 @@
+import logging
+from datetime import datetime
+
+import pandas as pd
+import pyspark.sql.functions as F
+
+from dsgrid.dimension.time import TimeZone
+from dsgrid.dimension.time import TimeZone, make_time_range
+from dsgrid.exceptions import DSGInvalidDataset
+from .dimensions import DateTimeDimensionModel
+from .time_dimension_base_config import TimeDimensionBaseConfig
+
+
+logger = logging.getLogger(__name__)
+
+
+class DateTimeDimensionConfig(TimeDimensionBaseConfig):
+    """Provides an interface to a DateTimeDimensionModel."""
```

Comment:
Why is this config needed? How does it relate to annual time? What would happen if we needed to add another type of time representation?

Does it really need its own file or should it also be in dimension_config.py?

Reply:
This was resolved in Dan's earlier PR